lnbits/core/services.py

```diff
@@ -502,6 +521,23 @@ def fee_reserve(amount_msat: int) -> int:
     return max(int(reserve_min), int(amount_msat * reserve_percent / 100.0))
 
 
+def service_fee(amount_msat: int) -> int:
+    service_fee_percent = settings.lnbits_service_fee
```

Comment:
The comment in `.env.example` says that "1.0" is "100% fee", so the fee is not specified in percent.

We need to either fix the code or the comment.